src/experimental/safe_html_transform/tests/test_filter.py

```diff
@@ -24,36 +24,41 @@ def setUp(self):
         )
 
     def test_url_end(self):
-        self.browser.open(self.portal_url + '/@@safe_html_transform-settings')
+        self.browser.open(self.portal_url + '/@@filter-controlpanel')
         self.assertTrue(
-            self.browser.url.endswith('safe_html_transform-settings'))
+            self.browser.url.endswith('filter-controlpanel'))
 
     def test_save_button(self):
-        self.browser.open(self.portal_url + '/@@safe_html_transform-settings')
+        self.browser.open(self.portal_url + '/@@filter-controlpanel')
         self.browser.getControl(name="form.buttons.save").click()
         self.assertTrue(
-            self.browser.url.endswith('safe_html_transform-settings'))
+            self.browser.url.endswith('filter-controlpanel'))
 
     def test_status_msg_after_save(self):
-        self.browser.open(self.portal_url + '/@@safe_html_transform-settings')
+        self.browser.open(self.portal_url + '/@@filter-controlpanel')
         self.browser.getControl(name="form.buttons.save").click()
         self.assertTrue(
             'Changes saved.' in self.browser.contents)
 
     def test_cancel_button(self):
-        self.browser.open(self.portal_url + '/@@safe_html_transform-settings')
+        self.browser.open(self.portal_url + '/@@filter-controlpanel')
         self.browser.getControl(name="form.buttons.cancel").click()
         self.assertTrue(
             self.browser.url.endswith('plone_control_panel'))
 
     def test_status_msg_after_cancel(self):
-        self.browser.open(self.portal_url + '/@@safe_html_transform-settings')
+        self.browser.open(self.portal_url + '/@@filter-controlpanel')
         self.browser.getControl(name="form.buttons.cancel").click()
         self.assertTrue(
             'Changes canceled.' in self.browser.contents)
 
+    def test_login(self):
```

Comment:
This test does not belong here, experimental.safe_html_transform has nothing to do with logins, right? :-)